resources/syntax/HTML (Twig).sublime-syntax

```diff
@@ -1,7 +1,7 @@
 %YAML 1.2
 ---
 name: HTML (Twig)
-scope: text.html.twig
+scope: text.html.twig meta.template.twig
```

Comment:
Main scope should not be a selector. That might cause trouble.

Instead add a customized main scope:

https://github.com/Sublime-Instincts/BetterJinja/blob/cac57e212dfdada3d79229b6f5cc5d920a1f90aa/Syntaxes/HTML/HTML%20(Jinja).sublime-syntax#L30-L32